README.md

```diff
@@ -18,16 +18,17 @@ cargo build
 Our Continuous Integration (CI) pipeline is implemented on top of
```

Comment:
Hmm, the readme should be updated as well now that we use the rust-vmm-ci directly. In other repositories we just linked the rust-vmm-ci and got rid of all the other details.

Reply:
Done :heavy_check_mark: